containers/conformance.md

```diff
@@ -0,0 +1,235 @@
+# Zowe Conformance Criteria Related to Containerization
+
+**DRAFT**
+
+These conformance criteria are applicable for all Zowe components intending to run in a containerized environment. The containerized environment could be Kubernetes or OpenShift running on Linux or Linux on Z. This may also apply to `docker-compose` running on Linux, Windows, Mac OS, or zCX.
+
+## Image
+
+In general, the image should follow [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). Below requirements are in addition to the list.
+
+### Base Image
+
+You are free to choose a base image based on your requirements.
+
+Here are our recommendations for core Zowe components:
+
+- Zowe base images:
+  * base on Ubuntu and Red Hat Universal Base Image,
+  * provide common dependencies including JDK and/or node.js,
+  * support both `amd64` and `s390x` architecture.
+- [Red Hat Universal Base Image 8 Minimal](https://developers.redhat.com/articles/ubi-faq?redirect_fragment=resources#ubi_details)
+- [Ubuntu](https://hub.docker.com/_/ubuntu)
+
+The image should contain as few software packages as possible for security, and should be as small as possible such as by reducing package count and layers.
```

Comment:
Make more specific requirements on base images:
- must be bash
- xargs, etc
- what about (o)ping, (o)netstat, (chtag), etc. - may make them all pre-reqs for base images they choose.